index.md

```diff
@@ -153,7 +153,7 @@
 [Cemu](https://cemu.info/) :c-pc: - Most accurate Wii U Emulator available.  
 [Decaf](https://github.com/decaf-emu/decaf-emu) :c-windows: :c-linux: - Alternative to Cemu, although not as advanced as the counterpart.
 
-[Ryujinx](https://ryujinx.org/) :c-pc: - Nintendo Switch emulator that aims to provide perfomance  
+[Ryujinx](https://github.com/GreemDev/Ryujinx/) :c-pc: - QOL fork of Nintendo Switch emulator that aims to provide perfomance  
```

Comment:
Rename the title to "GreenDev's fork of Ryujinx", and remove the QOL part. I would suggest to write "Fork of Ryujinx" at the end of the description, because the end user may not know the context behind the fork unless looking at the project's page directly, which is not the best UX for the user

Reply:
idk why i put qol

Reply:
I ask the same thing

Reply:
?